mod/homepage/template/self/homepage_top_banner.haml

```diff
@@ -6,8 +6,7 @@
         = nest :wikirate_white_logo, view: :core, size: :medium
       .col-md-10
         %span.wikirate-logo-title
-          Wiki
-          %strong Rate
+          Wiki<strong>Rate</strong>
```

Comment:
fixes unwanted space

Reply:
```
Wiki
%strong> Rate
```
[whitespace removal](http://haml.info/docs/yardoc/file.REFERENCE.html#whitespace-removal--and-)